CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/visualisation/TableVisualisation.java

```diff
@@ -107,16 +104,14 @@ public void populateTable(final List<C> items) {
 
         final FilteredList<C> filteredData = new FilteredList<>(tableData, predicate -> true);
         filteredData.addListener((Change<? extends C> change) -> table.refresh());
-        tableFilter.textProperty().addListener((observable, oldValue, newValue) -> {
-            filteredData.setPredicate(item -> {
-                if (StringUtils.isBlank(newValue)) {
-                    return true;
-                }
+        tableFilter.textProperty().addListener((observable, oldValue, newValue) -> filteredData.setPredicate(item -> {
+            if (StringUtils.isBlank(newValue)) {
```

Comment:
I think this can be simplified to be 
`tableFilter.textProperty().addListener((observable, oldValue, newValue) -> filteredData.setPredicate(item -> StringUtils.isBlank(newValue))`